go-apps/meep-rnis/sbi/rnis-sbi.go

```diff
@@ -175,18 +175,31 @@ func processActiveScenarioUpdate() {
 					mnc := ""
 					mcc := ""
 					cellId := ""
+					erabIdValid := false
 					if domain.CellularDomainConfig != nil {
 						mnc = domain.CellularDomainConfig.Mnc
 						mcc = domain.CellularDomainConfig.Mcc
+						cellId = domain.CellularDomainConfig.DefaultCellId
 					}
-					if poa.Poa4GConfig != nil {
-						if poa.Poa4GConfig.CellId != "" {
-							cellId = poa.Poa4GConfig.CellId
-						} else {
-							cellId = domain.CellularDomainConfig.DefaultCellId
+					switch poa.Type_ {
+					case mod.NodeTypePoa4G:
```

Comment:
What happens in the rest of the code if the scenario does not have a CellId set?

should use what is set else set a well-known hardcoded default value (ex. 0000000 or FFFFFFF)
same applies for 5G.

Reply:
cellId is set to a default value if a default value is provided in the operator cell element if it is not defined in the 4G or 5G poa config. If it is not set anywhere, it is not set and current services, like RNIS will not consider this as a valid poa (because a cellId is necessary). If cellId is not set, it is an empty string which is used when checking subscription and notification will be sent accordingly. No cellId implies the POA-GENERIC behaviour, not a 3gpp or wifi element so no support for the services.

Reply:
These kind of implications makes the code hard to follow: "No cellId implies the POA-GENERIC behaviour" 
How is someone supposed to know that from reading the code?

Reply:
its actually no cellId implies non 4G or 5G behavior, so everything else. Including Wifi but wifi has its own case, while generic POA falls into the default case....